developer_setup.md

```diff
@@ -193,14 +193,14 @@
 ```bash
 git clone git@github.com:JoeSmith/manageiq.git # Use "-o my_fork" if you don't want the remote to be named origin
 cd manageiq
-git remote add upstream git@github.com:ManageIQ/manageiq.git
+git remote add upstream https://github.com/ManageIQ/manageiq.git
```

Comment:
SSH is for pushing to your fork. I think adding upstream as `https` is good, because you'll probably never want to push to upstream. But you can always fetch from `https`